openedx/features/pakx/lms/pakx_admin_app/serializers.py

```diff
@@ -130,6 +131,13 @@ def get_completed_courses(obj):
 
 
 class CoursesSerializer(serializers.ModelSerializer):
+
+    instructor = serializers.SerializerMethodField()
+
     class Meta:
         model = CourseOverview
-        fields = ('display_name', 'start_date', 'end_date')
+        fields = ('display_name', 'instructor', 'start_date', 'end_date', 'course_image_url')
+
+    @staticmethod
+    def get_instructor(obj):
+        return CourseInstructorRole(obj.id).users_with_role().values_list('username', flat=True)
```

Comment:
please check how many queries this API is executing, it shouldn't be N times instead it should in constants

Reply:
<img width="1185" alt="Screenshot 2021-07-13 at 6 38 44 PM" src="https://user-images.githubusercontent.com/79907987/125461729-01435bd6-1f26-4826-98cf-e24629fc3814.png">